### DISCUSSION TITLE
docs/overview

### DISCUSSION THREAD

**Discussion by giscus** [casbin/casbin#1090] [Docs comments]
# docs/overview

Casbin overview

https://casbin.io/docs/overview

**Comment by buffolander** [comment]
Typo `senerios`

**Reply by hsluoyz** [reply]
@buffolander thanks!

@yoyo314 plz fix it

**Comment by jiey2** [comment]
We would like to present our special thanks to @hsluoyz for maintaining this repo.

**Reply by hsluoyz** [reply]
@jiey2 thanks!
